released/packages/coq-flocq-quickchick/coq-flocq-quickchick.1.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+name: "coq-flocq-quickchick"
+version: "1.0.0"
```

Comment:
I think the fields `name` and `version` should be removed as they are redundant with the file name.